osm2streets/src/lib.rs

```diff
@@ -200,40 +199,18 @@ impl StreetNetwork {
         self.intersections[&i].roads.clone()
     }
 
-    /// (Intersection polygon, polygons for roads, list of labeled polygons to debug)
-    #[allow(clippy::type_complexity)]
```

Comment:
Used in the abst rawmap editor. I'll do something different there -- probably running the transformation first and using geometry directly. Might have to come back later and add a method to just focus that transform on a few intersections.